test/query_tests/code_coverage_by_suite.test

```diff
@@ -0,0 +1,63 @@
+query: code_coverage_by_suite
+
+debug: False
+
+mock_data:
+  header:
+    - run.suite.fullname
+    - file
+    - source.file.name
+  meta:
+    content_type: application/json
+    format: table
+  data:
+    - - 
+    - 
+    - 0
```

Comment:
Is this the query that isn't working? Let's just leave broken queries out of it for now.

Reply:
Actually, this one runs, but doesn't return "data". It returns all the other keys though. The ones I listed above don't run at all - they each have the errors listed. Does this happen to you too?